tests/integration/tests/test_sanity_checks.py

```diff
@@ -151,3 +152,26 @@ def test_cspm_gcp_findings(cspm_client, match_type):
 
     results = get_findings(cspm_client, GCP_CONFIG_TIMEOUT, query, sort, match_type)
     assert len(results) > 0, f"The resource type '{match_type}' is missing"
+
+
+@pytest.mark.sanity
+@pytest.mark.parametrize("match_type", tests_data["cis_azure"])
+def test_cspm_azure_findings(cspm_client, match_type):
+    """
+    Test case to check for Azure findings in CSPM.
+
+    Args:
+        cspm_client: The elastic client object.
+        match_type (str): The resource type to match.
+
+    Returns:
+        None
+
+    Raises:
+        AssertionError: If the resource type is missing.
+    """
+    query_list = [{"term": {"rule.benchmark.id": "cis_azure"}}]
```

Comment:
@jeniawhite, the query is currently designed to search based on the `benchmark id` only. However, in your definition of `tests_data` for `cis_azure`, you specified `configuration`, and it appears that this term `{"term": {"resource.type": match_type}}` is not included in the query list.

Reply:
That was on purpose due to the fact that our cloud subscription is not static and might change.
In order to remove flakiness or having static resources, we are querying to make sure that we get any findings.
Once we will have a static subscription for the CI, we will introduce checks for every subtype.

Reply:
Is there a specific task or issue where we can document the absence of these tests for future reference?

Reply:
- https://github.com/elastic/cloudbeat/issues/1521